<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QWidget" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>I18N_EDITORWINDOW</string>
  </property>
  <layout class="QVBoxLayout" name="lay_editor_window">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="wid_main" native="true">
     <layout class="QHBoxLayout" name="lay_main">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wid_status" native="true">
     <layout class="QHBoxLayout" name="lay_status">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wid_buttons" native="true">
        <layout class="QHBoxLayout" name="lay_buttons">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btn_smartfont">
           <property name="toolTip">
            <string>I18N_SMARTFONT_TT</string>
           </property>
           <property name="text">
            <string>I18N_SMARTFONT</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_wordwrap">
           <property name="toolTip">
            <string>I18N_WORDWRAP_TT</string>
           </property>
           <property name="text">
            <string>I18N_WORDWRAP</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_linenumbers">
           <property name="toolTip">
            <string>I18N_LINENUMBERS_TT</string>
           </property>
           <property name="text">
            <string>I18N_LINENUMBERS</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_linehighlight">
           <property name="toolTip">
            <string>I18N_LINEHIGHLIGHT_TT</string>
           </property>
           <property name="text">
            <string>I18N_LINEHIGHLIGHT</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_tab">
           <property name="toolTip">
            <string>I18N_TAB_TT</string>
           </property>
           <property name="text">
            <string>I18N_TAB</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="btn_tabwidth">
           <property name="toolTip">
            <string>I18N_TABWIDTH_TT</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_syntax">
           <property name="text">
            <string>I18N_SYNTAX</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="box_syntax">
           <property name="maxVisibleItems">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="box_subsyntax">
           <property name="maxVisibleItems">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="spa_panel">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lab_pos">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_lines">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
